package.json

```diff
@@ -45,9 +45,17 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
+    "fast-crc32c": "^2.0.0",
     "google-gax": "^2.9.2"
   },
+  "rules": {
+    "node/no-extraneous-require": ["error", {
```

Comment:
this shouldn't be needed once you move the dependency to samples.

Reply:
Done. It did resolve the lint issue. Thanks!